src/librbd/cache/pwl/LogMap.cc

```diff
@@ -250,7 +250,7 @@ LogMapEntries<T> LogMap<T>::find_map_entries_locked(const BlockExtent &block_ext
  * represent the entire block extent of the GenericWriteLogEntry, and the
  * WriteLogMapEntry is added to the set.
  *
- * The set must not contain overlapping WriteLogMapEntrys. WriteLogMapEntrys
+ * The set must not contain overlapping WriteLogMapEntries. WriteLogMapEntries
```

Comment:
I think this spelling was intended.  I'd suggest the following:
```suggestion
 * The set must not contain overlapping write log entries. Entries
```